myapi.js

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
what's the point to `myapi.js` ?

Reply:
myapi.js  is just a file, I was playing around to test my understanding.